src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/HearingDefendantEntity.java

```diff
@@ -56,9 +46,9 @@ public class HearingDefendantEntity extends BaseImmutableEntity implements Seria
     @Column(name = "DEFENDANT_ID", nullable = false)
     private final String defendantId;
```

Comment:
Don't think we need this anymore?

Reply:
Changes not covered and are not critical are captured here and this is one of them https://dsdmoj.atlassian.net/browse/PIC-2622